api/src/main/java/uk/gov/hmcts/payment/api/controllers/FeesController.java

```diff
@@ -4,6 +4,7 @@
 import io.swagger.v3.oas.annotations.responses.ApiResponse;
 import io.swagger.v3.oas.annotations.responses.ApiResponses;
 import io.swagger.v3.oas.annotations.tags.Tag;
+import org.jetbrains.annotations.NotNull;
```

Comment:
I would suggest to use some other "NotNull", not from jetbrains. Maybe from jakarta.validation?